scripts/install.sh

```diff
@@ -110,7 +110,7 @@ check_dotnet_runtime_present() {
     fi
     #if dotnet exists, check the version required for b2k and install it.
     dotnetruntimes=$(dotnet --list-runtimes)
-    if [[ -z "${dotnetruntimes}" || ! "${dotnetruntimes}" =~ '6.0'* ]]; then
+    if [[ -z "${dotnetruntimes}" ]] || ! ([[ "${dotnetruntimes}" =~ '7.0'* ]] && [[ "${dotnetruntimes}" =~ 'AspNetCore'* ]]); then
```

Comment:
☕️🥷 this looks good, ❤️ just one thought to share - have we thought of any use case where there is a dependency on 6.0 which could lead to any complication of this script runs again? (In an already installed machine) ie.. what will be a behaviour in case I re-run this against my machine if I have B2K previously installed? 

Thank you so much for this.

Reply:
@Tatsinnit Thanks for the comments, yep, I have tested it and it installs dotnet 7.0 sdk in your machine and downloads b2k. Tested different scenarios like no dotnet installed, dotnet 6.0 installed etc... and everything works fine. 
do you think we should add such scenarios in the integration script tests action [here](https://github.com/Azure/Bridge-To-Kubernetes/actions/workflows/integration-check-install.yml) ? I can check on it but might be complicated because GH hosted runners come with dotnet 7.0 pre-installed already. Please let me know your thoughts on this.

Reply:
Sounds great, at-least we can go ahead with these changes since it’s tested. Thanks heaps.